xml/security_ldap.xml

```diff
@@ -510,8 +510,11 @@ tls_cacertdir = /etc/dirsrv/slapd-localhost/ <co xml:id="co.ldap.server.dsrc.rem
     </callout>
     <callout arearefs="co.ldap.server.dsrc.remote.cert">
      <para>
-      <remark>taroth 2019-03-07: @wbrown: is the following correct?</remark>
-      For this to work, copy <filename>/etc/dirsrv/slapd-localhost/ca.crt</filename>
+      If you have configured a certificate yourself, provide the path to the
+      CA certificate here (for remote access with TLS). If you do not have
+      configured a certificate yourself, the server generates a self-signed
+      certificate which you can trust. In that case, specify
```

Comment:
"in that case ..." could be better as "It can be found at `location`. Copy it to a readable location or to the client machine you are using ds*  from"